lib.docs/src/strato/rollup.config.js

```diff
@@ -44,29 +30,14 @@ export default async function getConfig() {
       sourcemap: true,
     } ],
     plugins: [
-      rollupImportMapPlugin({
-        imports: {
-          "bignumber.js": "https://unpkg.com/bignumber.js@9.0.2/bignumber.mjs"
-        }
-      }),
-      webWorkerLoader({
-        sourcemap: true,
-      }),
-      alias({
-        entries: [
-          { find: 'dotenv', replacement: getPathOf("./polyfills/dotenv.js") },
-          { find: /.*SolidityCompiler.*/, replacement: getPathOf("./polyfills/SolidityCompiler.js") },
-          { find: /.*StratoLogger.*/, replacement: getPathOf("./polyfills/StratoLogger.js") },
-
-          // Alias-ing browser specific implementations
-          { find: /(.*encoding\/Hex.*)/, replacement: '$1.native' }
-        ],
+      strato({ 

```

Comment:
This is how using the plugin will look like. We'll have a separate demo-repo to highlight this better.